### ISSUE TITLE
Cannot access private folder after key reset due to "MDServer Unauthorised" error

### ISSUE THREAD

**Issue by froggypad** [keybase/client#15588]
After resetting my keys, I am unable to access my private folder through any device. The app gives this error:
![screenshot 2019-01-17 at 21 45 30](https://user-images.githubusercontent.com/3658668/51350995-88483080-1aa1-11e9-8a89-3fa3a8bdb3be.png)

I backed up the folder's contents and deleted them from the folder before resetting, so no data is lost, but I can't use this feature.

my log id: c76393ef63178a8a1c16771c

**Comment by maxtaco** [comment]
That error isn't ideal, but your data is not recoverable after a reset (as we warn you).  If it were, then we could decrypt it.

cc @strib  / @songgao  on the error

**Comment by froggypad** [comment]
That fixed it, thanks for the quick response!

@strib I had added my other devices using a windows partition that's very old, and my understanding was that if I wanted to revoke it, it'd revoke all the other devices that were added using it(which I think I was wrong about, looking back). Once I got a new machine, I figured I should reset and use it to add my devices before I start actually using my account, so I wouldn't have to worry later.
